Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
> 15. $this, the element to which this type applies is the same exact instance as the current class in the given context. As such this type is a stricter version of static as, in addition, the returned instance must not only be of the same class but also the same instance.

This type is often used as return value for methods implementing the Fluent Interface design pattern.

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md